Common/FileUtil.cpp

```diff
@@ -93,7 +93,7 @@ FILE *OpenCFile(const std::string &filename, const char *mode)
 
 bool OpenCPPFile(std::fstream & stream, const std::string &filename, std::ios::openmode mode)
 {
-#if defined(_WIN32) && defined(UNICODE)
+#if defined(_WIN32) && defined(UNICODE) && !defined(__MINGW32__)
```

Comment:
Hm, why is this needed? No wstring support in MingW32's libraries?

(ditto for the other places)

Reply:
It's odd yeah - I am even compiling with UNICODE defines.

Reply:
This is the error I get when I take out && !defined(__MINGW32__) here -

https://hastebin.com/bucebodoci.vbs